test/Swashbuckle.AspNetCore.IntegrationTests/CustomDocumentSerializerTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Extensions.ApiDescriptions;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Readers;
+using Swashbuckle.AspNetCore.Swagger;
+using Xunit;
```

Comment:
This is a lot of usings - I feel some of them are redundant and can be deleted.

Reply:
I did a sort + clean on them